### PR TITLE
Add support for App Protect DoS

### PR DESCRIPTION
**Pull request by lucacome**
Adds support for App Protect DoS

### CODE REVIEW

File: docs/nginx-ingress-controller.md [hunk 1]
```diff
@@ -153,4 +153,14 @@ spec:
 
 | Field | Type | Description | Required |
 | --- | --- | --- | --- |
-| `enable` | `boolean` | Enable App Protect. | Yes |
+| `enable` | `boolean` | Enable App Protect WAF. | Yes |
+
+## NginxIngressController.AppProtectDos
```
Comment (pleshakov) [inline docs/nginx-ingress-controller.md:158]: Could you add a row about appProtectDos field in the main table and link to this section? This is for consistency, as we document all other second-level fields this way.

**Review by ciarams87 — APPROVED** [review]
Looks good once Michael's feedback is addressed 👍🏼

File: docs/nginx-ingress-controller.md (hunk 1 above)
Comment (lucacome) [reply docs/nginx-ingress-controller.md:159]: added

**Review by pleshakov — APPROVED** [review]
👍

**Pull request merged**